hardware_isolation_entry.cpp

```diff
@@ -38,6 +38,15 @@ Entry::Entry(sdbusplus::bus::bus& bus, const std::string& objPath,
     this->emit_object_added();
 }
 
+void Entry::delete_()
+{
+    if (!resolved())
```

Comment:
Need to look system state before deleting to avoid clash with host?

Reply:
ah missed, good catch. Maybe I need to look type and decide? for example, only for manual delete?